lib/rspec/core/formatters.rb

```diff
@@ -203,8 +203,8 @@ def built_in_formatter(key)
         ProgressFormatter
       when 'j', 'json'
         JsonFormatter
-      when 'bisect'
```

Comment:
is this a change to the CLI API?

Reply:
It's a change to a private part of the CLI API.  `--format bisect` has never been listed in the `--help` and isn't usable as a standalone formatter.  In fact, if you run `rspec --format bisect`, it crashes:

``` ruby
➜  rspec-core git:(master) bin/rspec --format bisect
/Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:823:in `parse_uri': can't parse uri:druby://localhost: (DRb::DRbBadURI)
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:889:in `uri_option'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:785:in `block in uri_option'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:783:in `each'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:783:in `uri_option'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:1067:in `initialize'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:1047:in `new'
	from /Users/myron/.rubies/ruby-2.1.7/lib/ruby/2.1.0/drb/drb.rb:1047:in `new_with_uri'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/formatters/bisect_formatter.rb:19:in `initialize'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/formatters.rb:148:in `new'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/formatters.rb:148:in `add'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/configuration.rb:877:in `add_formatter'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/configuration_options.rb:117:in `block in load_formatters_into'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/configuration_options.rb:117:in `each'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/configuration_options.rb:117:in `load_formatters_into'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/configuration_options.rb:23:in `configure'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/runner.rb:99:in `setup'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/runner.rb:86:in `run'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/runner.rb:71:in `run'
	from /Users/myron/code/rspec-dev/repos/rspec-core/lib/rspec/core/runner.rb:45:in `invoke'
	from /Users/myron/code/rspec-dev/repos/rspec-core/exe/rspec:4:in `<top (required)>'
	from bin/rspec:14:in `load'
	from bin/rspec:14:in `<main>'
```

This "formatter" only exists for when users use `rspec --bisect`.  In that case, the main bisect process boots a DRb server, and then adds `--format bisect` to each shell command it generates to run a subset of the suite, so that the specific test run can communicate the results back to the main process.

Since it's a private API, I don't think we should call it out in the changelog.